src/middleware/LoadAppealMiddleware.ts

```diff
@@ -21,41 +22,41 @@ export class LoadAppealMiddleware extends BaseMiddleware {
     constructor(@inject(AppealsService) private readonly appealsService: AppealsService) {
         super();
     }
-    // @ts-ignore
-    public async handler(req: Request, res: Response, next: NextFunction): Promise<void> {
+    public handler: RequestHandler = makeAsyncRequestHandler(
+        // @ts-ignore
+        async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+            const companyNumber = req.query[COMPANY_NUMBER_QUERY_KEY] as string;
```

Comment:
if there is no company number or appealId?

Reply:
Check below
```
 try {
            new SchemaValidator(companyNumberSchema).validate(companyNumber);
       } catch (err) {
            throw new Error('Tried to load appeal from an invalid company number');
        }
// and
 if (appealId && !applicationData.appeal) 
```

Reply:
Those cases are covered :). If there is no appealId then the file restrictions middleware will take it as the user is still completing the appeal

Reply:
ah i see, thanks for clarifying 👍